### ISSUE TITLE
Missing migration cms/0019 in pypi package v3.5.1

### ISSUE THREAD

**Issue by alexandrenorman** [divio/django-cms#6300]
### Summary

Missing migration 0019 for cms in pypi package v3.5.1
migration has been included in 0018… but not published yet on pypi

```
# -*- coding: utf-8 -*-
# Generated by Django 1.10.8 on 2018-03-06 16:01
from __future__ import unicode_literals

import django.contrib.auth.models
from django.db import migrations


class Migration(migrations.Migration):

    dependencies = [
        ('cms', '0018_pagenode'),
    ]

    operations = [
        migrations.AlterModelManagers(
            name='pageusergroup',
            managers=[
                ('objects', django.contrib.auth.models.GroupManager()),
            ],
        ),
    ]                  
```

### Environment

* Python version: 3.6
* Django version: 3.11
* django CMS version: 3.5.1

**Comment by czpython** [comment]
Hello @alexandrenorman,
Thanks for reporting this.
Can you run: `pip show django-cms` ?
We added the missing operation to migration 0018 which is released under 3.5.1

**Comment by alexandrenorman** [comment]
False report in fact, I mixed between two docker containers… 
the missing migration was still running under v3.5.0. 

v3.5.1 is correct ! Sorry !!
